front/src/components/ModalDelete.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+
+    <div class="">
+        <vodal :show="show" animation="rotate" @hide="hideModal">
+            <p>¿Esta seguro que quiere eliminar este evento?</p>
+            <button @click="deleteEvent">SI</button>
+            <button @click="hideModal">NO</button>
+        </vodal>
+    </div>
+</template>
+
+<script>
+import Vue from 'vue';
+import Vodal from 'vodal';
+import axios from 'axios';
+
+Vue.component(Vodal.name, Vodal);
+
+export default {
+    name: 'app',
+    props: {
+        show: Boolean,
+        id: String
+    },
+    data(){
+        return{
+            showComp: this.show
+        }
+    },
+    methods:{
+        hideModal(){
+            this.showComp = false;
+            this.updateCoachStatus();
+        },
+        updateCoachStatus: function(event){
+            this.$emit("hideModal", this.showComp) ;
+        },
+        deleteEvent(){
+            this.$emit("refrestList") ;
+/*             axios.delete("http://localhost:3000/api/v1/deleteEvent/"+id)
```

Comment:
We specified as a bad practice to leave code comented on PR. It's better to delete this entire code or uncomment it and end the functionality but if you think that this cant be finished yet and is really important to have this comment here, leave it like that

Reply:
this will be fixed in the branch fixingEventList